docs/faq.md

```diff
@@ -202,6 +202,60 @@ jib.from.image = 'custom-base-image'
 
 We currently support adding a custom directory with an **incubating** feature. This feature may change in later versions. If your application needs to use custom files, place them into the `src/main/jib` folder. Files placed here will be added to the filesystem of the container. For example, `src/main/jib/foo/bar` would add `/foo/bar` into the container filesystem.
 
+### <a id="i-want-more-control-over-which-files-to-place-in-the-extra-directory"></a>I want more control over which files to place in the extra directory/where they end up in the container
```

Comment:
Is the anchor here necessary? I think we should be fine to use a shorter bookmark link if GitHub generates a shorter one.

Reply:
https://github.com/GoogleContainerTools/jib/pull/1092#discussion_r222408669